### ISSUE TITLE
Ability to Drag and Drop files into Insert Files

### ISSUE THREAD

**Issue by lucasisaza** [VenusInterns/BlogTemplate#99]

**Comment by dmcfaddengalway** [comment]
I'd like to take on this issue, let me k ow if I can help.

**Comment by umaslakshmi** [comment]
@dmcfaddengalway Check out my branch File-Uploads if you want an idea of how we are saving the files. Right now we don't have the drag and drop UI so that would be great to have. Thanks for expressing interest!

**Comment by dmcfaddengalway** [comment]
@umaslakshmi absolutely. This will be the first time I worked with drag and drop but know its a part of HTML now so will see what I can do and if I have trouble I'll ask. Thanks! This is my second OSS Contrib so I'm new to this.
